Source/DafnyTestGeneration/BlockBasedModification.cs

```diff
@@ -31,7 +31,7 @@ public class BlockBasedModification : ProgramModification {
       string? line;
       var stringReader = new StringReader(log);
       var newBlocksCovered = false;
-      while ((line = stringReader.ReadLine()) != null) {
+      while ((line = await stringReader.ReadLineAsync()) != null) {
```

Comment:
Is this useful / necessary?

Reply:
In general it improves performance. Rider presents it as a warning.